customizations/README.md

```diff
@@ -1414,3 +1416,24 @@ REVISION	UPDATED                 	STATUS         	CHART             	APP VERSION
 29      	Tue Oct  4 10:58:22 2022	deployed       	carto-1.42.10-beta	2022.9.28  	Rollback to 27
 ``` 
 Now you can run the upgrade operation again
+
+### 413 Request Entity Too Large
+
+  You are trying to make a POST request to the SQL Api with a large payload. Please see the following considerations:
+
+  - We support a payload up to 10Mb
+
+  - If your payload is lower than 10Mb, probably the error code is returned by a service in a higher layer than the Carto Selfhosted environment. Please upload your service configuration to be able to manage higher requests. E.g:
+
+  In Nginx, you have to add this config line to your location service:
```

Comment:
```suggestion
  In Nginx, you have to add this config line to your location configuration:
```

Reply:
I think we should explain this better, the use case of an `nginx ingress` is clear, but i'm not sure if the users will understand the other use case "`nginx`". Probably we need to elaborate this better